data/Vaaran Bhai Gurdas Ji/15.json

```diff
@@ -2179,7 +2179,7 @@
         "id": "C3K5",
         "source_page": 15,
         "source_line": 12,
-        "gurmukhi": "pwirjwqu guru Cif kY; mMgin klpqroˆ Pl kcy [",
+        "gurmukhi": "pwirjwqu guru Cif kY; mMgin klp qroˆ Pl kcy [",
```

Comment:
Need reference

Reply:
Should be `qroN`

Reply:
This line should not have been changed in this commit/PR. Fixed with another commit.